frontend/app/module/polling_stations/PollingStationsLayout.tsx

```diff
@@ -0,0 +1,23 @@
+import { Outlet } from "react-router-dom";
+
+import { Footer } from "app/component/footer/Footer";
+import { NavBar } from "app/component/navbar/NavBar";
+
+import { PollingStationListProvider } from "@kiesraad/api";
+import { PageTitle } from "@kiesraad/ui";
+import { useNumericParam } from "@kiesraad/util";
+
+export function PollingStationsLayout() {
+  const electionId = useNumericParam("electionId");
+
+  return (
+    <div className="app-layout">
+      <PageTitle title="Overzicht verkiezingen - Abacus" />
+      <PollingStationListProvider electionId={electionId}>
+        <NavBar />
+        <Outlet />
+        <Footer />
```

Comment:
There's a double footer on the page, see https://f174b725.kiesraad-abacus.pages.dev/elections/1/polling-stations

Might be because of the `<Footer />` in the `OverviewLayout`?